src/node/nodestate.h

```diff
@@ -383,6 +383,9 @@ namespace ccf
           // public ledger has been read
           accept_member_connections();
 
+          // Accept node connections for operators to check the state of the network
+          accept_node_connections();
```

Comment:
This was necessary since the management and node frontends have been merged. Otherwise, during the recovery of the public ledger, it is not possible for an operator to query the state of the node.